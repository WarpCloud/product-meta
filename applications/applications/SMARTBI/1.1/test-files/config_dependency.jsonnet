{
  application_type: "SMARTBI",
  application_name: "smartbi1",
  application_version: "1.1",
  user_config: {},
}
