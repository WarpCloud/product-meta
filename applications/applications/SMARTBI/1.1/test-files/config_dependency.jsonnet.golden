{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "smartbi1",
            "application_type": "SMARTBI"
         },
         "configs": {
            "smartbi_mysql_cpu_limit": 2,
            "smartbi_mysql_cpu_request": 0.10000000000000001,
            "smartbi_mysql_memory_limit": 4,
            "smartbi_mysql_memory_request": 1,
            "smartbi_tomcat_cpu_limit": 2,
            "smartbi_tomcat_cpu_request": 0.10000000000000001,
            "smartbi_tomcat_memory_limit": 4,
            "smartbi_tomcat_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "smartbi",
         "name": "smartbi1-smartbi",
         "version": "1.1"
      }
   ],
   "instance_settings": { },
   "version": ""
}
